spring-cloud-dataflow-tasklauncher/spring-cloud-dataflow-tasklauncher-sink/pom.xml

```diff
@@ -0,0 +1,352 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+		 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.springframework.cloud</groupId>
+		<artifactId>spring-cloud-dataflow-tasklauncher</artifactId>
+		<version>2.9.0-SNAPSHOT</version>
+	</parent>
+	<artifactId>spring-cloud-dataflow-tasklauncher-sink</artifactId>
+	<properties>
+		<revision>3.1.0-SNAPSHOT</revision>
+		<stream-apps-core.version>${revision}</stream-apps-core.version>
+		<java-functions.version>1.1.0-SNAPSHOT</java-functions.version>
+		<apps.base-image>springcloud/baseimage:1.0.0</apps.base-image>
+		<mockserver.version>5.10</mockserver.version>
+		<spring-cloud-stream-dependencies.version>3.1.4-SNAPSHOT</spring-cloud-stream-dependencies.version>
+		<spring-cloud-stream.version>3.1.4-SNAPSHOT</spring-cloud-stream.version>
+		<spring-cloud-dataflow-apps-generator-plugin.version>1.0.3</spring-cloud-dataflow-apps-generator-plugin.version>
+		<spring-cloud-dataflow-apps-docs-plugin.version>1.0.3</spring-cloud-dataflow-apps-docs-plugin.version>
+		<spring-cloud-dataflow-apps-metadata-plugin.version>1.0.3</spring-cloud-dataflow-apps-metadata-plugin.version>
+		<java-cfenv-boot.version>2.1.2.RELEASE</java-cfenv-boot.version>
+		<prometheus-rsocket.version>1.3.0</prometheus-rsocket.version>
+		<!-- Boot 2.4.x needs wavefront-spring-boot version 2.1.0-RC1+ -->
+		<wavefront-spring-boot.version>2.2.0</wavefront-spring-boot.version>
+		<spring-cloud.version>2020.0.4-SNAPSHOT</spring-cloud.version>
```

Comment:
So, this really need to come from https://github.com/spring-cloud/spring-cloud-dataflow-build/blob/d5aa7c2f7ed50fa705890a2364236a078fe03dfe/spring-cloud-dataflow-build-dependencies/pom.xml#L21. Hope we don't get into too much trouble as dataflow has different lifecycle as spring-cloud which some of other dependant libs rely on(as apps release have been atop of spring-cloud and never been dependant what we do on dataflow).

Reply:
> Regarding registration and use of this migrated task launcher, it'd be best to handle that outside of this PR as moving tasklauncher to dataflow is enough complex on its own. Lets first properly create a jar/image and then handle even more complex part of using it separately.
> 
> Having said that, what we did for ctr app is not that nice and there's a lessons to learn from it. We chose to put ctr app into server config(which defaults to maven coordinates) and on k8s space that need to be image. On some level it'd be nice to have a new app registration subsystem/type where app version is managed by dataflow itself.

Fair enough, I have reverted the registry changes in this PR